pkgs/development/python-modules/rst2pdf/default.nix

```diff
@@ -63,7 +63,7 @@ buildPythonPackage rec {
 
   postInstall = ''
     mkdir -p $man/share/man/man1/
-    ${docutils}/bin/rst2man.py doc/rst2pdf.rst $man/share/man/man1/rst2pdf.1
+    ${docutils}/bin/rst2man doc/rst2pdf.rst $man/share/man/man1/rst2pdf.1
```

Comment:
I would suggest a more Nixpkgs way to do this:

```bash
${docutils}/bin/rst2man doc/rst2pdf.rst rst2pdf.1
installManPage rst2pdf.1
```

Reply:
Updated!